lib/ui/screens/update_contact_screen.dart

```diff
@@ -155,6 +168,8 @@ class _UpdateContactScreenState extends State<UpdateContactScreen> {
                     ),
                 ),
             ),
-        ); 
+        );
+        return NullScreen(message: "Cannot edit cause this contact doesn't exist",
```

Comment:
Should not use nested Scaffold (Scaffold inside another Scaffold).
Remove NullScreen class, instead, build a widget here for empty message is enough